pom.xml

```diff
@@ -66,10 +67,20 @@
             <!-- handle broken builds on jdk1.8 due to doclint function -->
             <id>default-jdk18-profile</id>
             <activation>
-                <jdk>[1.8,)</jdk>
+                <jdk>1.8</jdk>
```

Comment:
I tried this notation but it was not working with my latest maven installed. Have you checked the profile to be working?